core/controllers/topic_editor.py

```diff
@@ -97,12 +97,16 @@ def post(self, topic_id):
         thumbnail_filename = self.payload.get('filename')
         thumbnail_bg_color = self.payload.get('thumbnailBgColor')
         raw_image = self.request.get('image')
+        url_fragment = self.payload.get('url_fragment')
```

Comment:
Maybe be clearer in the naming about what type of url fragment this is (topic/story/etc.)